src/lib/babylon/scene/coordinates.ts

```diff
@@ -0,0 +1,15 @@
+import { Vector3 } from "@babylonjs/core";
+import { SceneContext } from "./context";
+
+// This function converts a "global" coordinate to a "scene" coordinate system
+export function globalCoordinatesToSceneCoordinates(scene: SceneContext, coordinates: Vector3) {
```

Comment:
_Naming suggestion_
Could be shortened to `globalToSceneCoordinates`, or even `globalToSceneVector` if that's appropriate.